.circleci/config.yml

```diff
@@ -502,10 +502,8 @@ jobs:
             nvm deactivate
             echo "node --version: $(node --version)"
             echo "npm --version:  $(npm --version)"
-            # Install JsThemis locally
-            rm -rf node_modules package-lock.json
-            make jsthemis_install
             # Run JsThemis tests
+            rm -rf src/wrappers/themis/jsthemis/node_modules
```

Comment:
can we call something like `make jsthemis_uninstall`?

Reply:
> can we call something like make jsthemis_uninstall?

My goal was to avoid doing that specifically, but it does not seem to be working. So my current plan is to revert the last commit which messes with jsthemis_install usage during tests, as soon as I figure out the way to install dev dependencies of JsThemis into node_modules in the source tree root along with JsThemis itself. Then call `jsthemis_install` to clear our compiled code before switching to a different Node.js.